spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,36 @@
+require 'beaker-rspec'
+require 'beaker-puppet'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_module
+install_module_dependencies
+
+RSpec.configure do |c|
+
+  # Configure all nodes in nodeset
+  c.before :suite do
+    hosts.each do |host|
+      case host[:platform]
+      when /debian-7-amd64|debian-8-amd64|ubuntu-16.04-amd64/
+        on host, puppet('module', 'install', 'puppetlabs-apt')
```

Comment:
you can do something like this:
```ruby
install_module_from_forge('puppetlabs-apt', '>= 4.1.0 < 7.0.0') if fact('os.family') == 'Debian'
```

Reply:
I'll use function for `epel` module.
And this  comment makes me realize that `Debian` family OSes does not need special apt sources.